lib/order.rb

```diff
@@ -0,0 +1,84 @@
+require_relative 'text'
+
+class Order
+
+  attr_reader :number, :basket, :total, :history
+
+  def initialize
+    @number = 1
```

Comment:
Unique order number can help to track orders which will make things so much easier for the takeaway